File: quests/global/items/CHRMGold.pl
sub EVENT_SCALE_CALC {
  my $money = $client->GetCarriedMoney();

  if($money < 0) {
    $money = 0;
  }
  if($money > 64425000) {
     $money = 64425000
  }

  $questitem->SetScale($money/64425000);
}

File: global/items/CHRMGold.lua
-- CHRMGold

function event_scale_calc(e)
    local money = e.owner:GetCarriedMoney() / 1000;

    if(money < 0) then
        money = 0;
    end

    if(money > 21475) then
        money = 21475;
    end

    e.self:SetScale(money/21475);
end